GitUI/GitUICommands.cs

```diff
@@ -1914,5 +1919,10 @@ internal TestAccessor(GitUICommands commands)
 
             internal bool RunCommandBasedOnArgument(string[] args) => _commands.RunCommandBasedOnArgument(args, InitializeArguments(args));
         }
+
+        private bool RunDotCommand()
+        {
+            return StartBrowseDialog();
+        }
```

Comment:
Please remove. I do not see any benefit of adding an alias. Furthermore "." is rather a path than a command.

Reply:
Agree. No alias, add a comment that this is interpreted as a path
Also add a test, similar to RunCommandBasedOnArgument_browse()

Reply:
Sure. I have added the test case `RunCommandBasedOnArgument_dot()`.
Thank you much for the sample reference.
The changes will be in the next pull.
CC: @mstv @gerhardol